src/main/scala/it/pagopa/interop/backendforfrontend/server/impl/dependencies/Dependencies.scala

```diff
@@ -100,7 +100,20 @@ trait Dependencies {
     )
     .toFuture
 
-  def makeController(jwtReader: JWTReader, blockingEc: ExecutionContextExecutor)(implicit
+  def fileManager(blockingEc: ExecutionContextExecutor): FileManager =
+    FileManager.get(ApplicationConfiguration.storageKind match {
+      case "S3"   => FileManager.S3
+      case "file" => FileManager.File
+      case _      => throw new Exception("Incorrect File Manager")
+    })(blockingEc)
+
+  def getWhiteList(blockingEc: ExecutionContextExecutor): Future[List[String]] = {
+    fileManager(blockingEc)
+      .get(ApplicationConfiguration.storageContainer)("")
```

Comment:
⚠️ We're missing the path here!

Reply:
Also, consider the eventuality of the file missing and use a sane fallback like a `Future.successful(Nil)`

Reply:
@galales I want to hear your opinion here 👆. This attitude may hide further problems. Is it maybe worth failing fast?

Reply:
good point 🤔 

Defaulting to `Nil` is not a security threat, as it would block every non-IPA organization anyway.
On the other hand, our organization must always be in the allow list, even in dev env, so maybe failing fast could be more useful to us as an alarm of "you forgot something".

Of course if we leave the file empty we are back to square one, but that's on us 🤣 
I would go for the second approach

Reply:
I suggested to @bloodsky the first approach, but after the @galales' comment, I agree with him. Fail fast is the best choice.